tests/lib/core-config.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
```

Comment:
Is everything in this file a part a public function? Can you please add a comment about what each function is expected to do please.

I'm somewhat wary of another shell module that is sourced as those tend to be more fragile than executed programs. For instance, the interpreter here is "bash" but that's just a comment that is entirely ignored when this file is sourced from a non-bash shell program.

I'm also somewhat worried about naming, all the functions have free-form names. Shell does not issue any warning when a function is re-defined. I think we should either use a program that lives in `tests/lib/tools` and is invoked by path OR have a well-defined naming scheme as debugging a failure like that will not be fun.

Reply:
@zyga, Agree regarding to the function names, I'll add function comments and update names